test/e2e/functional_test.go

```diff
@@ -1237,3 +1240,51 @@ func (s *FunctionalSuite) TestControllerMetrics() {
 	assert.NoError(s.T(), err)
 	assert.Equal(s.T(), http.StatusOK, resp.StatusCode)
 }
+
+func (s *FunctionalSuite) TestRolloutPauseDurationGreaterThanProgressDeadlineSeconds() {
+	s.Given().
+		HealthyRollout(`
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollout-canary
+spec:
+  replicas: 3
+  progressDeadlineSeconds: 5
+  selector:
+    matchLabels:
+      app: rollout-canary
+  template:
+    metadata:
+      labels:
+        app: rollout-canary
+    spec:
+      containers:
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
```

Comment:
Can we use the nginx image like we do for the rest of the e2e tests? It will speed up tests since it should already be loaded into the kubernetes environment.

Also, `imagePullPolicy: Always` will slow down tests, and the container port for nginx is 80